keras_cv/layers/regularization/drop_path.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+
+
+@tf.keras.utils.register_keras_serializable(package="keras_cv")
+class DropPath(tf.keras.layers.Layer):
+    """
+    Implements the DropPath layer. Some samples from the batch are randomly 
+    dropped during training with dropping probability `drop_rate`. Note that 
+    this layer DOES drop individual samples and not the entire batch. 
```

Comment:
"Note that this layer drops individual samples within a batch and not the entire batch."